### PR TITLE
R doc 1805 static js ts index

### PR DESCRIPTION
**Pull request by morhilai**

### CODE REVIEW

File: Documentation/5.1/Samples/csharp/Raven.Documentation.Samples/DocumentExtensions/TimeSeries/Indexing.cs [hunk 1]
```diff
@@ -108,6 +108,24 @@ select new
         }
         #endregion
 
+        #region indexes_AbstractJavaScriptCreationTask
+        public class Company_TradeVolume_ByDate : AbstractJavaScriptTimeSeriesIndexCreationTask
+        {
+            public Company_TradeVolume_ByDate()
+            {
+                Maps = new HashSet<string> { @"
+                    timeSeries.map('Companies', 'StockPrices', function (ts) {
```
Comment (ppekrol) [inline Documentation/5.1/Samples/csharp/Raven.Documentation.Samples/DocumentExtensions/TimeSeries/Indexing.cs:117]: Have we written somewhere what is available under 'ts' ? What methods? What fields?

File: Documentation/5.1/Samples/csharp/Raven.Documentation.Samples/DocumentExtensions/TimeSeries/Indexing.cs [hunk 2]
```diff
@@ -108,6 +108,24 @@ select new
...
+                        return ts.Entries.map(entry => ({
```
Comment (ppekrol) [inline Documentation/5.1/Samples/csharp/Raven.Documentation.Samples/DocumentExtensions/TimeSeries/Indexing.cs:118]: Have we written somewhere what is available under 'entry' ? What methods? What fields?

File: Documentation/5.1/Samples/csharp/Raven.Documentation.Samples/DocumentExtensions/TimeSeries/Indexing.cs (hunk 1 above)
Comment (morhilai) [reply Documentation/5.1/Samples/csharp/Raven.Documentation.Samples/DocumentExtensions/TimeSeries/Indexing.cs:117]: There's this: https://ravendb.net/docs/article-page/5.0/csharp/document-extensions/timeseries/indexing#timeseriessegment-object
As far as methods I have found these undocumented methods that don't seem relevant to indexing:
>TimeSeriesEntry<T> method:
`public void Deconstruct(out DateTime timestamp, out T value, out string tag)`
And this conversion method:
`public static TimeSeriesRollupEntry<T> AsRollupEntry<T>(this TimeSeriesEntry<T> entry)`

I will document them on the pages [Named Time Series Values](https://ravendb.net/docs/article-page/5.0/csharp/document-extensions/timeseries/client-api/named-time-series-values) and [Rollup and Retention](https://ravendb.net/docs/article-page/5.0/csharp/document-extensions/timeseries/rollup-and-retention). Are there other methods or fields you're thinking of?

**Comment by ppekrol** [comment]
LGTM @morhilai can you rebase?

**Comment by morhilai** [comment]
> LGTM @morhilai can you rebase?

Done

**Pull request merged**